docs/getting-started/install-endpoint.asciidoc

```diff
@@ -30,8 +30,32 @@ image::images/install-endpoint/endpoint-cloud-sec-integrations-page.png[Search r
 image::images/install-endpoint/endpoint-cloud-security-configuration.png[Add {elastic-defend} integration page,75%]
 +
 . Configure the {elastic-defend} integration with an **Integration name** and optional **Description**.
+. Select the type of environment you want to protect, either *Traditional Endpoints* or *Cloud Workloads*.
+. Next, select a configuration preset. Each preset comes with different default settings for {agent} — you can further customize these later. To learn more, refer to {fleet-guide}/agent-policy.html[{agent} policies].
++
+For Traditional Endpoints, there are three presets:
++
+* *Next Generation Antivirus:*
+  - Events collected: Process
+  - Active preventions: Malware, Ransomware, Memory Protection
+* *Essential Endpoint Detection & Response:*
+  - Events collected: Process, Network, File
+  - Active preventions: Malware, Ransomware, Memory Protection
+* *Complete Endpoint Detection & Response:*
+  - Events collected: Process, Network, File, Session Data
+  - Active preventions: Malware, Ransomware, Memory Protection
++
+For Cloud Workloads, there are two presets:
++
+* *All events:*
+  - Events collected: Process, Network, File, Session Data
```

Comment:
also, I know you have process, network, file, session data as separate items. But truthfully session data is additive to the process, network and file events. Session Data does stand on it's own.
We want to make that clear to the user that session data is an enrichment of the existing events